### PR TITLE
Add Exercise 13 files

### PR DESCRIPTION
**Pull request by Tayoto-Mikaella**
Please review my code. Thank you

### CODE REVIEW

**Comment by Villa-VillaDiana** [comment]
This is noted. I’ll start reviewing your codes after I review Paul’s code.

**Comment by Villa-VillaDiana** [comment]
I just finished reviewing the code of the other group. I will start reviewing yours.

File: tayoto_mikaellaantonette/index.html [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+                </div>
+                <div>
+                  <button id="sort_asc">Sort by Date (ASC)</button>
+                  <button id="sort_desc">Sort by Date (DESC)</button>
+                </div>
+                <ul id="comment_list"></ul>
+              </section>
+            </div>
+          </form>
+          <script src="./script.js"></script>
```
Comment (Villa-VillaDiana) [inline tayoto_mikaellaantonette/index.html]: Please put your script tag at the end of your code

`    <script src="./script.js"></script>`
`  </body>`
`</html>`

File: tayoto_mikaellaantonette/index.html [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+          <div>
+            <h2>Comments on my goals</h2>
+            <ul>
+              <li>Goodluck! - Sofia</li>
+              <li>Great Job! - Eliza</li>
+              <li>
+                Focused and embrace the challenges that come your way -Precious
+              </li>
+            </ul>
+            <li>Praying for your journey! - Alona</li>
```
Comment (Villa-VillaDiana) [inline tayoto_mikaellaantonette/index.html:112]: Her comment is not included in the list

**Review by Villa-VillaDiana — CHANGES_REQUESTED** [review]
Before I continue reviewing your codes, kindly fix your comment button because it is currently not working.

File: tayoto_mikaellaantonette/script.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  const name = nameInput.value.trim();
+  const comment = commentInput.value.trim();
+  const date = new Date();
+  if (!name || !comment) {
+    return;
+  }
+
+  const newComment = document.createElement("li");
+  newComment.innerHTML = `<strong>${name}</strong> - ${comment} - 
+  <time datetime="${date}">${date.toLocaleString()}</time>`;
```
Comment (Villa-VillaDiana) [inline tayoto_mikaellaantonette/script.js]: Observe proper indentation. Please follow this format:

```
  newComment.innerHTML = `<strong>${name}</strong> - ${comment} - 
    <time datetime="${date}">${date.toLocaleString()}</time>`;
```

File: tayoto_mikaellaantonette/script.js [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+function comment() {
+  const name = nameInput.value.trim();
+  const comment = commentInput.value.trim();
+
+  if (name || comment) {
+    form.querySelector('button[type="submit"]').disabled = false;
+  } else {
+    form.querySelector('button[type="submit"]').disabled = true;
+  }
+}
```
Comment (Villa-VillaDiana) [inline tayoto_mikaellaantonette/script.js]: If you are not using this function, I recommend deleting this.

**Review by Villa-VillaDiana — CHANGES_REQUESTED** [review]

**Review by Villa-VillaDiana — APPROVED** [review]
Looks good to me, you can now merge this :))

**Comment by Tayoto-Mikaella** [comment]
Thank you! ^^

**Pull request merged**